sbnanaobj/StandardRecord/SRPfoChar.cxx

```diff
@@ -0,0 +1,30 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRPfoChar.cxx
+// \brief   SRPfoChar object for Pfo Characterisation (track vs shower) features (MVA inputs).
+// \author  howard@fnal.gov
+////////////////////////////////////////////////////////////////////////
+
+#include "sbnanaobj/StandardRecord/SRPfoChar.h"
+
+namespace caf
+{
+  SRPfoChar::SRPfoChar()
```

Comment:
Can we capitalize as `SRPFOChar` please?

Reply:
Done and pushed for the class name/constructor. Should I make the change for the file name as well?

Reply:
Yes please

Reply:
Done!